Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Controls/Controls Toolkit/KryptonComboBoxEnhanced.cs

```diff
@@ -0,0 +1,298 @@
+﻿using System;
+using System.ComponentModel;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Krypton.Toolkit.Suite.Extended.Controls
+{
+    [ToolboxBitmap(typeof(ComboBox))]
+    public class KryptonComboBoxEnhanced : ComboBox
```

Comment:
Why is this not derived from the KryptonComboBox, so that all the Theme handling is done once !

Reply:
Deleted in subsequent !